jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/vsdm/VsdmDepositorThreaded.java

```diff
@@ -0,0 +1,227 @@
+package us.dot.its.jpo.ode.vsdm;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetSocketAddress;
+import java.net.SocketException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.oss.asn1.AbstractData;
+import com.oss.asn1.Coder;
+import com.oss.asn1.EncodeFailedException;
+import com.oss.asn1.EncodeNotSupportedException;
+
+import gov.usdot.asn1.j2735.CVSampleMessageBuilder;
+import gov.usdot.asn1.j2735.CVTypeHelper;
+import gov.usdot.asn1.j2735.J2735Util;
+import us.dot.its.jpo.ode.j2735.J2735;
+import us.dot.its.jpo.ode.j2735.semi.ServiceRequest;
+import us.dot.its.jpo.ode.j2735.semi.ServiceResponse;
+import us.dot.its.jpo.ode.j2735.semi.VehSitDataMessage;
+import us.dot.its.jpo.ode.j2735.semi.VsmType;
+
+public class VsdmDepositorThreaded implements Runnable{
+
+	@Override
+	public void run() {
+		String targetHost = "104.130.170.234";
+		int targetPort = 46753;
+		
+		int vsdmSenderPort = 6666;
+		VsdmSender vsdmSender = new VsdmSender(targetHost, targetPort, vsdmSenderPort);
+		Thread vsdmSenderThread = new Thread(vsdmSender, "VsdmSenderThread");
+		vsdmSenderThread.start();
+		
+		int srSenderPort = 5556;
```

Comment:
targetHost, targetPort, targetPort and srSenderPort should be in OdeProperties. 

Does srSenderPort and vsdmSenderPort have to be different?

Suggested names for the properties: something like sdcHost, sdcPort, odeVsdDepositorPort, odeVsdListenerPort